draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -1609,23 +1609,22 @@ user agent MUST process the cookie as follows:
     creation-time and the last-access-time to the current date and time.
 
 6.  If the cookie-attribute-list contains an attribute with an attribute-name
-    of "Max-Age":
+    of "Max-Age" or an attribute with an attribute-name of "Expires":
 
     1.  Set the cookie's persistent-flag to true.
 
-    2.  Set the cookie's expiry-time to attribute-value of the last
-        attribute in the cookie-attribute-list with an attribute-name of
-        "Max-Age".
+    2.  If the cookie-attribute-list contains an attribute with an attribute-name
+    of "Max-Age":
 
-    Otherwise, if the cookie-attribute-list contains an attribute with an
-    attribute-name of "Expires" (and does not contain an attribute with an
-    attribute-name of "Max-Age"):
+        1. Set the cookie's expiry-time to attribute-value of the last
+           attribute in the cookie-attribute-list with an attribute-name of
+           "Max-Age".
 
-    1.  Set the cookie's persistent-flag to true.
+        Otherwise:
 
-    2.  Set the cookie's expiry-time to attribute-value of the last
-        attribute in the cookie-attribute-list with an attribute-name of
-        "Expires".
+        1. Set the cookie's expiry-time to attribute-value of the last
+           attribute in the cookie-attribute-list with an attribute-name of
+           "Expires".
 
```

Comment:
> Section 5:  (Recognizing that this is from the original RFC) The nested
numbered lists are difficult to parse.  For example Section 5.7, #6 has 3 sets
of sub numbered lists that appear to be distinct.  If these sub numbered lists
are necessary (and when there is merely a #1 without a #2, one might argue it
isn't 'necessary') then perhaps characters other than numbers might be clearer.

I changed the conditionals around in a way that, imo, is easier to read. Unfortunately it still has 3 levels of nesting. I'm fine with keeping, reverting, or doing something else to this, open to feedback.

Reply:
> Switching between numbers and letters would make the algorithm more difficult for me to understand than sticking with a numbered list of steps throughout.

Agreed

>  I think any change here must be non-normative

Agreed

> I don't mind nesting, but, **as above**, we could drop the nesting for single-step lists pretty easily

Maybe I'm missing it, but what are you referring to with "as above"?